components/Signin.tsx

```diff
@@ -28,9 +33,12 @@ const SignIn = () => {
       if(!res.ok){
         const {error} = await res.json();
         toast.error(error || "Email validation failed");
+        setTimeout(() => {setIsButtonLocked(false)}, 2000);
```

Comment:
```suggestion
        setTimeout(() => {setIsButtonLocked(false)}, 120000);
```

 I think it's fine to put the timer for 2mins instead of 2s :)

Reply:
Sure bhaiya, will update it:)